scripts/test.ts

```diff
@@ -171,6 +171,11 @@ export default function (args: ParsedArgs, logger: logging.Logger) {
     runner.env.addReporter(new IstanbulReporter());
   }
 
+  if (args.large) {
+    // Default timeout for large specs is 5 minutes.
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 150000;
```

Comment:
5 minutes seems fairly long.  Are there tests that will take that long?

Reply:
The biggest timeout before was 120s, and I think some of the timeouts I left are actually 180s now. 

Neither goes up to 5mins though, my comment is just wrong.